### DISCUSSION TITLE
scrum prep Milestone 1 - 2

### DISCUSSION THREAD

**Discussion by Yahid1** [uprm-inso4116-2024-2025-s1/semester-project-trolley-tracker-app#17] [General]
Write your scrum here.

**Comment by dAninAd1248** [comment]
I created two documentation tasks, ran the existing code, read the lecture presentations, and brainstormed ideas for a lecture topic task.

**Comment by AngelMartinez33** [comment]
Held a weekly meeting to update my team on the latest news and make sure everyone is on the same page. Created documentation tasks.

**Comment by MrL9000** [comment]
Met with both the entire team and just the Metrics team regarding the state of the app and reminding everyone of the work that must be done. Admittedly I was quite busy with other classes, and because of it I didn't have much time to work on my tasks. Thankfully, nearing the end of this week I'm finally able to properly focus on the projcect. I have already worked on most of the design of the app, and, after that, I'll move on to the other doccumentation and lecutre topic tasks.

**Comment by Brandell13** [comment]
This week I have been creating my tasks and the Lecture topic tasks. Additionally, I have been running the code to make sure everything is working correctly.

**Comment by AlexisTraversoPR** [comment]
This week I was not able to do much but I Created a documentation task and a task for a notifications feature. I also assigned myself a task to create a figma for the menu of the app. I started looking into possible designs for the menu and getting reacquainted with figma.

**Comment by Yahid1** [comment]
This week I ensured guidelines and task were made, organized meetings and showed the project. Had a RUMBus team member as contact to answer any questions about the project. Currently working on creating a doc explaining how to run artificial backend data.

**Comment by IvanierB** [comment]
This week I worked mainly on a documentation task that focused on defining the terminology for the project. I also attended a meeting with the whole team and watched/learned how to run the app for RumBus.

**Comment by Janiel777** [comment]
I'm having a meeting today, Friday, September 13th, to plan how to divide up the Notifications Team's tasks to create issues for the next milestone. I've been working on my two documentation issues and I'm almost done with them. I've been mostly keeping my group active and notifying them of what they have to do.

**Comment by marcosarraiza** [comment]
This past week I created about 3 tasks, including one lecture topic task and started working on them. I also familiarized myself with RUMbus's existing code so I can ve ready to add to it whenever needed.

**Comment by dylanDeJesus21** [comment]
Created a documentation issue and an issue to generate a general flowchart of the program. Started creating a playlist of Angular 18 and Typescript frameworks. Finally, also set up of the RUMbus's code.

**Comment by sebasagir** [comment]
I began this week by starting to learn the basics of AngularJS. then I chose to spend the majority of the week familiarizing myself with the big picture concepts of the RumBus project, such that I have an easier time with documentation, since that it what we will mainly work this milestone. I also ran the app once the repo was updated to confirm it was working on my machine

**Comment by diegorobles2** [comment]
I created my documentation tasks, and made sure the code was running well on my computer

**Reply by juan-ortiz92** [reply]
I have been assigned to the notifications implementation and have started working on the issue for documentation regarding the domain and behavior and events.

**Comment by AleDiazz** [comment]
This week, I have been selecting the tasks I'll be working on, along with the lecture topics I’ll be focusing on. Additionally, I’ve been reviewing and familiarizing myself with the code and language to start coding once we have the opportunity to work on it.

**Comment by juan-ortiz92** [comment]
I have been assigned to the notifications implementation and have started working on the issue for documentation regarding the domain and behavior and events.

**Comment by warm007** [comment]
During the beginning of this week, I read through the presentations and started brainstorming for the lecture topic task. Then, I attended the general meeting. But unfortunately, I couldn't attend the team meeting. I had a hell week, so I can't be very attentive. I still need tasks to complete for this milestone. For today's presential meeting, I hope to have two documentation tasks and complete them over the weekend.

**Comment by JaseyAcevedo** [comment]
Worked on my documentation task downloaded dependancies for proyect formulating posible lecture topic tasks

**Comment by JoelTorres20** [comment]
This week I've been working on my documentation tasks and have finished setting up and running the base of the app. I have also continued to learn angular.js.

**Comment by rafa-quiles** [comment]
Not much this week, researched documentation

**Comment by JConcepcion01** [comment]
I created two documentation tasks and have been working on them, plus attending the general team meeting

**Comment by yamayrie** [comment]
This week I created two documentation tasks, I attended a general group meeting, and worked on setting up everything to be able to run the project. I'll be working on 2.2.1 and 2.2.2 for the documentation. I started reading the document on what is needed to properly do these parts and started brainstorming what I'll be putting down.
